<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(94, 92, 100);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID CARD TAG</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="readRFID">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Total">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>451</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Sign up</string>
    </property>
    <widget class="QPushButton" name="btnJoin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>410</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(119, 118, 123);</string>
     </property>
     <property name="title">
      <string>Car Information</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Car Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Kind of Car</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbKind">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputCarNum">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>251</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(119, 118, 123);</string>
     </property>
     <property name="title">
      <string>Member Information</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputPhone">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>410</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>초기화</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>870</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUID">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <zorder>groupBox_Total</zorder>
   <zorder>label_2</zorder>
   <zorder>readRFID</zorder>
   <zorder>btnExit</zorder>
   <zorder>btnUID</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
